src/ResourceManager/Sql/Commands.Sql.Test/packages.config

```diff
@@ -6,7 +6,7 @@
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Graph.RBAC" version="3.2.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.0.1-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Sql" version="1.13.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="2.4.0-preview" targetFramework="net45" />
```

Comment:
Please remove/revert changes to the files after this: 
src/StackAdmin/AzureRM/AzureRM.psm1
src/StackAdmin/AzureStack/AzureStack.psm1
tools/AzureRM/AzureRM.psm1
tools/LocalFeed/Microsoft.Azure.Management.Network.19.0.1-preview.nupkg
tools/LocalFeed/Microsoft.Azure.Management.Network.19.0.1-preview.symbols.nupkg

Reply:
Ping - these files need to be reverted or deleted before the PR is accepted.

Reply:
I shall do these after all changes are accepted. As doing these will cause the build to break.